### PR TITLE
RavenDB-5700 Fix how expiration bundle is handled by smuggler

### PR DESCRIPTION
**Pull request by IdanHaim**
* Add Test to expiration bundle

### CODE REVIEW

File: test/FastTests/Smuggler/SmugglerApiTests.cs [hunk 1]
```diff
@@ -109,6 +115,51 @@ select new
...
+                        metadata[Constants.Expiration.RavenExpirationDate] =
+                            new RavenJValue(DateTime.UtcNow.AddSeconds(10)
+                                .ToString(Default.DateTimeOffsetFormatsToWrite));
+
+                        await session.SaveChangesAsync().ConfigureAwait(false);
+                    }
+
+                    var date = DateTime.UtcNow;
+
+                    SpinWait.SpinUntil(()=> date.AddSeconds(10) < date , TimeSpan.FromSeconds(10));
```
Comment (ml054) [inline test/FastTests/Smuggler/SmugglerApiTests.cs:140]: What this line do?

**Comment by IdanHaim** [comment]
This will make sure we wait until we have expired document. 10 seconds in
this case.

**Comment by ml054** [comment]
I think 
>   await Task.Delay(10000);

Might be less resource intensive.

**Comment by IdanHaim** [comment]
I think you right Task.Delay will take less resources

**Pull request merged**